recipes/prefect/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "prefect" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3bcf015244ebaa6250fb27f755282f9f1569caf73114b5cdc31e861922d986f5
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - click >=7.0,<8.0
+    - cloudpickle >=0.6.0,<0.7
+    - croniter >=0.3,<0.4
+    - cryptography >=2.2.2,<3.0
+    - dask >=0.18,<2.0
+    - distributed >=1.21.8,<2.0
+    - docker-py >=3.4.1,<4.0
+    - idna >=2.5,<2.8
+    - marshmallow >=3.0.0b19,<3.1
+    - marshmallow-oneofschema >=2.0.0b2,<3.0
+    - mypy >=0.600,<0.700
+    - mypy_extensions >=0.4.0,<0.5
+    - pendulum >=2.0.4,<3.0
+    - python-dateutil >2.7.3,<3.0
+    - pyyaml >=3.13,<4.3
+    - pytz >=2018.7
+    - requests >=2.20,<3.0
+    - toml >=0.9.4,<1.0
+    - typing >=3.6.4,<4.0
+    - typing_extensions >=3.6.4,<4.0
+    - python-xxhash >=1.2.0,<2.0
+
+test:
+  imports:
+    - prefect
+    - prefect.triggers
+    - prefect.schedules
+    - prefect.tasks.shell
+    - prefect.tasks.control_flow
+    - prefect.tasks.github
+    - prefect.tasks.templates
+    - prefect.tasks.templates.jinja2
+
+about:
+  home: https://github.com/PrefectHQ/prefect
+  license: APACHE-2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: 'The Prefect Core automation engine '
+  description: |
+    Prefect is a new workflow management system, designed for modern
+    infrastructure and powered by the open-source Prefect Core workflow engine.
+    Users organize `Tasks` into `Flows`, and Prefect takes care of the rest.
+  doc_url: https://docs.prefect.io/
+
+extra:
+  recipe-maintainers:
+    - dhirschfeld
+    - rzillman
```

Comment:
@rzillman Are you also happy to be listed as a maintainer of this recipe?

Reply:
@chrisburr Yep